runtime/src/test_support/cpp/CompilerGenerated.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+#ifndef RUNTIME_COMPILER_GENERATED_H
+#define RUNTIME_COMPILER_GENERATED_H
+
+#define THROW_NOT_IMPLEMENTED throw std::runtime_error("Not implemented for tests");
```

Comment:
Maybe we can do without this macros? We can define a plain function `void ThrowNotImplementedForTests() { … }` here. However, I don't see anything wrong with repeating `throw std::runtime_error(...)` at each site either.

Reply:
Fixed by inlining the macro.